Source/VisualStudioTools/Private/BlueprintAssetHelper.cpp

```diff
@@ -5,6 +5,8 @@
 
 #include "AssetRegistry/AssetRegistryModule.h"
 #include "Engine/BlueprintGeneratedClass.h"
+#include "Engine/BlueprintCore.h"
```

Comment:
If possible, can you keep the includes sorted alphabetically? It's a readability style we'd like to follow.
This is the case for the other files too.

Reply:
New changes pushed that conform to this style below.